po/de.po

```diff
@@ -9,7 +9,7 @@ msgstr ""
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-01-28 20:40+0100\n"
 "PO-Revision-Date: 2017-08-23 13:02+1000\n"
-"Last-Translator: Peter Hutterer <peter.hutterer@who-t.net>\n"
+"Last-Translator: Nicolai Weitkemper <kontakt@nicolaiweitkemper.de>\n"
```

Comment:
Not sure if/how to update these.

Reply:
depends on whether you want the credit. these days we look at github more so than these files, but having a contact to someone who can do the translations is always good.

Reply:
Okay, I'll leave it in then. :)